app/models/document.rb

```diff
@@ -60,6 +60,16 @@ def self.at_slug(document_types, slug)
     find_by(document_type: document_types, slug:)
   end
 
+  def self.document_types
```

Comment:
Should this be called `edition_types`, since a Document can have multiple editions (all of a different document type)?

Reply:
I did wonder if this should belong on the `Edition` model, but then the `documents` table has a `document_type` column (which I suppose is just the latest edition's type, though I can't see that that's definitely the case from the model?). You'd know better than me though! Which of these three seems the most correct?

```rb
Edition.types # matches a database field
Document.document_types # matches a database field
Document.edition_types
```